### ISSUE TITLE
When generating Swagger docs for controllers, add option to specify those

### ISSUE THREAD

**Issue by RafalBDS** [nestjs/swagger#129]
## I'm submitting a...

<pre><code>
[ ] Regression 
[ ] Bug report
[x] Feature request
[ ] Documentation issue or request
[ ] Support request => Please do not submit support request here, instead post your question on Stack Overflow.
</code></pre>

## Current behavior

Currently, when generating Swagger docs via
`document = SwaggerModule.createDocument(app, options)`
Swagger-scanner maps all modules controllers, which result in documentation generated for whole app module

## Expected behavior

Documentation is generated only for particular controlers
`documentCats= SwaggerModule.createDocument(app, options, [CatCtrl])`
`documentDogs= SwaggerModule.createDocument(app, options, [DogsCtrl])`

## What is the motivation / use case for changing the behavior?

Example use case - API needs to be provided to two separate clients
```
app.use('/documentation/catsCtrl',  AUTH_MIDD, (req, res) => res.json(documentCats));
app.use('/documentation/dogsCtrl', AUTH_MIDD, (req, res) => res.json(documentDogs));

````

## Environment

<pre><code>
├─┬ @nestjs/common@5.0.0
├─┬ @nestjs/core@5.0.0
├─┬ @nestjs/swagger@2.0.0
├─┬ @nestjs/testing@5.0.0

**Comment by kamilmysliwiec** [comment]
#132 has been merged already (v2.5.0).
API:
```typescript
const document = SwaggerModule.createDocument(app, options, {
    include: [UsersModule],
});
```

**Comment by RafalBDS** [comment]
@kamilmysliwiec 
Fresh one ;). Thanks :)
